fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+const fizzbuzz = (x) => {
+        if ((x % 3 === 0)&& (x % 5 === 0)) {
+            return `fizzbuzz`;
```

Comment:
If you're just returning a plain string (no contatenation or interpolation) you should just use single quotes. Backticks (`) are used when you need to do interpolation with a template literal - e.g. `I love ${cat}`